documentation/docs/examples/java/create-channel.md

```diff
@@ -0,0 +1,67 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: The example-5 script authenticates an Integration Service client to manage Identities using the Admin identity created in example-0 and then creates a channel, writes data on the channel, reads data from the channel.
+keywords:
+- create channel
+- read channel
+- write channel
+- example
+---
+# Create Channel
+
+The [example-5](https://github.com/albydeca/iota-is-sdk/blob/main/examples/src/main/java/net/gradbase/examples/CreateChannel.java)
+script authenticates an Integration Service client to manage Identities using the Admin identity created in [example-0](how-to-run-examples) and then performs the following tasks:
```

Comment:
```suggestion
script authenticates an Integration Services client to manage identities using the admin identity created in [example-0](how-to-run-examples) and then performs the following tasks:
```

identity and admin identity: these are just technical terms, no need to capitalize technical terms. An Integration Services client: a client is a technical term, so non-capitalized, Integration Services is a name, so capitalized. Same across the rest of the contribution. No worries, this is a confusing rule.